### ISSUE TITLE
[feature]add export and import settings feature like raycast on MacOS

### ISSUE THREAD

**Issue by liaohui5** [Flow-Launcher/Flow.Launcher#2643]
**Is your feature request related to a problem? Please describe.**

add export and import settings feature like raycast on MacOS

**Comment by liaohui5** [comment]
> Even though we don't have an explicit feature for this, you can still copy the files within %appsata%/FlowLauncher and that would contains all settings or plugins you installed.

thank you for reply, but i hope add this feature in next major version :)

**Comment by onesounds** [comment]
@taooceros Wrong! There is only two answer to this request: "I will" or "I will later". "You can do it manually" is not a good answer. (Of course, I make the same mistake when my wife asks me to take out the trash.) As you know, Copying and backing up user settings manually is the way people in the Stone Age used to do it. People in that era used stone weapon and 💾. All modern software has this function.

If It's literally just copying a configuration file, but there should be a button to do it with a single click. We have a pretty complicated setting. We should at least have an export.

**Comment by Faeest** [comment]
Users are facing challenges when it comes to transferring their **settings** and **plugin** configurations across devices or creating backups, as the current manual process is time-consuming and prone to errors. **To address this**, we propose implementing an **export/import feature** that would allow users to easily save their settings and plugin configurations to a file, **import** them on another device, and be **prompted for necessary plugin installations** during the import process. This solution is preferable to alternatives like manual file copying, cloud syncing, or extensive documentation, as it offers a convenient and user-friendly approach. The feature would significantly enhance the user experience, particularly for power users and those who frequently assist others in setting up configurations, while also serving as an efficient backup solution.

**Comment by indigofairyx** [comment]
yes please and thank you! 

I had been running with the portable install for some months. With version 1.19 I did a regular install over the portable and it didn't move my user data folder, it erased it! Lucky I keep backups and was able to unzip\copy my User Data into its new location.
